tests/PackagePrivateLinterTest.hack

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\HHAST;
+
+final class PackagePrivateLinterTest extends TestCase {
+  use LinterTestTrait;
+
+  protected function getLinter(string $file): BaseLinter {
+    return PackagePrivateLinter::fromPath($file);
+  }
+
+  public function getCleanExamples(): vec<(string)> {
+    return vec[
+      tuple('<?hh function test();'),
```

Comment:
add tests in namespaces using private definitions

Reply:
Assuming you meant rewriting the tuple part here to include `private` namespace.

Reply:
adding another tuple to the same vec :)